openedx/core/djangoapps/oauth_dispatch/views.py

```diff
@@ -159,15 +156,9 @@ def _get_jwt_builder(self, user, is_client_restricted):
         # (asymmetric) key.
         # TODO: ARCH-162
         use_asymmetric_key = ENFORCE_JWT_SCOPES.is_enabled() and is_client_restricted
-        monitoring_utils.set_custom_metric('oauth_asymmetric_jwt', use_asymmetric_key)
-
-        log.info("Using Asymmetric JWT: %s", use_asymmetric_key)
-
         return JwtBuilder(
             user,
             asymmetric=use_asymmetric_key,
-            secret=settings.JWT_AUTH['JWT_SECRET_KEY'],
```

Comment:
Can you clarify why this was needed and no longer is needed?

Reply:
It is not needed since `JwtBuilder` defaults to this value when `secret` isn't passed in.  Unsure why it was needed earlier.